sql/init.sql

```diff
@@ -23,7 +23,7 @@ CREATE UNIQUE INDEX p2_val_idx ON p2 (val);
 CREATE INDEX p2_ununi_id_val_idx ON p2 (val);
 CREATE INDEX p2_val_idx_1 ON p2 USING hash (val);
 CREATE INDEX p2_val_id_idx ON p2 (val, id);
-CREATE INDEX p2_val_idx2 ON p2 (val COLLATE "ja_JP");
+CREATE INDEX p2_val_idx2 ON p2 (val COLLATE "C");
```

Comment:
Yes, I have found this to be rather annoying because it is not really portable. I think that we should change it separately, and will do so if there are no objections.